exercises/sum.c

```diff
@@ -0,0 +1,24 @@
+/*
+  Write a program that takes as input two numbers and print the sum.
+
+  Output:
+  Insert the first number: 1
+  Insert the second number: 2
+  Sum: 3
+*/
+
+#include <stdio.h>
+int x;
+int y;
+int sum;
```

Comment:
In questo caso ti suggerisco di definire le variabili localmente e non in maniera globale (nel main). Eventualmente dai anche dei nomi significativi alle variabili e non troppo generici: 
Esempio: `int first_number`